hipercam/scripts/fits2hcm.py

```diff
@@ -65,14 +65,20 @@ def fits2hcm(args=None):
            WHTCAM :
              Camera used by Richard Ashley in Feb 2021.
 
+           OSIRIS :
+             Imager and spectrograph on the GTC. I've only tested this with
+             data taken with the new detector known as OSIRIS+ and even then,
+             only with what I believe is the standard setup
+             (2x2 binning and full frame).
+

```

Comment:
Can we change this docstring to assign blame, and also the name of the instrument? So, something like

```
OSIRIS+ : 
     Imager and spectrograph on the GTC. Added by Alex Brown in Nov 2023. 
     Only tested with 2x2 binning and full frame
```